keda/Chart.yaml

```
@@ -4,7 +4,7 @@ description: Event-based autoscaler for workloads on Kubernetes
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 1.3.2
+version: 1.3.3
```

This should be 1.4.0 now, please merge master.
```suggestion
version: 1.4.0
```